README.md

```diff
@@ -119,8 +119,11 @@ Note: currently there is ongoing work on a “displayName” concept. When the a
 
 **[Where shall images (products or icons) of an asset be stored, e.g. for the use in dashboards?](#id31)** <a id="id31"></a><!-- ID: 31 -->
 
-Such images may be included in the “TechnicalData” submodel as File submodel elements. It is recommended to distinguish between product photos and functional icons. Product photos shall get the idShort = "productPicture", functional icons shall get the idShort = "functionalSymbol".
-(Answered: 2020-09-21)
+With V3.0RC01 of Part 1 assetInformation/defaultThumbnail was introduced. The type of these thumbnails is "File".
```

Comment:
@BirgitBoss is there only one defaultThumbnail per submodel? Is it supposed to show an "icon" or an "image/photo" of the asset?

Reply:
The thumbnail is not per submodel but per AAS (i.e. AssetAdministrationShell/assetInformation. There can be more than one thumbnail, cardinality 0..*. The Type is File so there is not distinction between icon and photo (as we proposed in our original answert): this might be a new requirement to the metamodel of the AAS.

Reply:
okay, thanks, no further comments from my side.